tests/VoTest/DateRangeTest.php

```diff
@@ -3,6 +3,7 @@
 namespace VoTest;
 
 use DateTime;
+use DateTimeImmutable;
```

Comment:
Immutable dates also need to be tested. There's a mix of mutable and immutable if you look at the tests, though it should be sufficient to test that the constructor, separately, provides an immutable date invariant.